packages/bot-web-ui/src/stores/app-store.js

```diff
@@ -236,8 +236,29 @@ export default class AppStore {
         }
     };
 
-    showDigitalOptionsMaltainvestError = (client, common) => {
+    showDigitalOptionsMaltainvestError = (client, common, ui) => {
+        const has_non_eu_account = Object.keys(client.accounts).some(acc => acc?.startsWith('CR'));
```

Comment:
couldn't use: `const is_cr = client?.standpoint?.svg`, it works asynchronously, brings me wrong value